template/hooks/php/cs-fixer/pre-commit

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+. "$GIT_DIR/hooks/change_detector.sh"
+
+if has_changed pre-commit ./**/*.php ./*.php
```

Comment:
I was confused when I saw this work in unit tests, but it turns out it does not in some cases. I have to rewrite this.